### PR TITLE
Update air bridge options

### PR DESCRIPTION
**Pull request by hskim881028**
SSIA

### CODE REVIEW

File: nekoyume/Assets/Airbridge/Resources/AirbridgeData.asset [hunk 1]
```diff
@@ -14,11 +14,11 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   appName: ninechronicles
   appToken: c476471c40544e479b367e2ddee1e587
-  logLevel: 0
+  logLevel: 2
   sdkSignatureSecretID: 
   sdkSignatureSecret: 
-  iOSURIScheme: https://apps.apple.com/kr/app/nine-chronicles-m/id6462364578
-  androidURIScheme: https://play.google.com/store/apps/details?id=com.planetariumlabs.ninechroniclesmobile
+  iOSURIScheme: ninechroniclesmobile
```
Comment (longfin) [inline nekoyume/Assets/Airbridge/Resources/AirbridgeData.asset:20]: Is this change intended? @hskim881028

Comment (hskim881028) [reply nekoyume/Assets/Airbridge/Resources/AirbridgeData.asset:20]: yes

Comment (longfin) [reply nekoyume/Assets/Airbridge/Resources/AirbridgeData.asset:20]: It seems so.

**Review by longfin — APPROVED** [review]
